source/client/process_impl.h

```diff
@@ -94,6 +94,16 @@ class ProcessImpl : public Process, public Envoy::Logger::Loggable<Envoy::Logger
 
   bool requestExecutionCancellation() override;
 
+  /**
+   * Initializes the User Defined Output Plugin factories that will be used in this process, stores
+   * them in this process, and returns a reference to them.
+   *
+   * @return a vector of pairs, with the TypedExtensionConfig and its corresponding factory.
+   */
+  std::vector<
```

Comment:
I'm probably just missing something, but I can't seem to find this function's definition.

Reply:
Thanks for catching this. You aren't missing anything. I missed cleaning this up when I moved this method to user_defined_output_factory_creator.